spec/features/workflow/unpublishing_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "spec_helper"
```

Comment:
We've setup the app so you don't need to require the spec_helper :-).

Reply:
Fixed up. The other test files have been cleaned up too.